recipes/args/all/test_package/.gitignore

```diff
@@ -0,0 +1 @@
+/build/
```

Comment:
This `.gitignore` is not necessary.
`build` folders are ignored by the root `.gitignore`.

https://github.com/conan-io/conan-center-index/blob/e00cb6ae6df4e2124cc3287c9b4df77f8311a1ed/.gitignore#L16

Reply:
Originally developed standalone, survived being copied into the fork...